### DISCUSSION TITLE
打开闻达后无法启用，没有读取到RTST记忆区default，将新建。 'NoneType' object has no attribute 'embedding_function'

### DISCUSSION THREAD

**Discussion by lojie012** [wenda-LLM/wenda#225] [Q&A]
使用的是4.29版本，GLM6B配置好了，但是进入页面端口后，每一次回复闻达都是回复省略号
运行GLM6B.bat,显示：
没有读取到RTST记忆区default，将新建。 'NoneType' object has no attribute 'embedding_function'

**Comment by lojie012** [comment]
![微信截图_20230502152210](https://user-images.githubusercontent.com/128771488/235604620-05eac30a-7751-4b8c-a668-64b7a2f40178.png)

**Reply by l15y** [reply]
报错是什么

**Reply by Capter121** [reply]
解决了吗？

**Reply by fishso** [reply]
win11 ，懒人包，同样的问题

**Reply by hejia7** [reply]
刚pull的代码，也是后端打印：
没有读取到RTST记忆区rtst_kf，将新建。
'NoneType' object has no attribute 'embedding_function'
没有读取到RTST记忆区default，将新建。
'NoneType' object has no attribute 'embedding_function'

**Reply by huanglongfei1** [reply]
plugins文件夹下的zhushiku_rtst.py 第106行的路径改成绝对memory的绝对路径就可以解决了
